charts/aws-vpc-cni/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 name: aws-vpc-cni
-version: 1.1.9
-appVersion: "v1.9.1"
+version: 1.1.11
```

Comment:
just curious, why is the chart version skipping a version to 1.1.11 when previous version is 1.1.9 ?

Reply:
Just needs a fetch/rebase :) - https://github.com/aws/amazon-vpc-cni-k8s/blob/master/charts/aws-vpc-cni/Chart.yaml#L3

Reply:
Yes - v1.9.3 chart update is missing in release-1.10 branch